### ISSUE TITLE
[Bug]: http://metrics.beam.apache.org/ is not responding

### ISSUE THREAD

**Issue by aaltay** [apache/beam#27675]
### What happened?

Metrics dashboards seems to be down.

Choosing IO connectors component. But I think we need an "infra" component. /cc @damccorm 

### Issue Priority

Priority: 3 (minor)

### Issue Components

- [ ] Component: Python SDK
- [ ] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [X] Component: IO connector
- [ ] Component: Beam examples
- [ ] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

**Comment by aaltay** [comment]
Tagging: @liferoad

**Comment by mosche** [comment]
This looks like a DNS issue, the dashboard works if using https://s.apache.org/beam-community-metrics

**Comment by liferoad** [comment]
Yes, indeed. We need to update the DNS with the new IP. @Abacn can you update this?

**Comment by Abacn** [comment]
Thanks for reporting, opened https://issues.apache.org/jira/browse/INFRA-24836

**Comment by aaltay** [comment]
This seems to be working again. Thank you for the quick fix!
